pyiron/sphinx/base.py

```diff
@@ -1242,6 +1243,42 @@ def write_input(self):
             f.write("include <parameters.sx>;\n\n")
             f.write(self.input.sphinx.to_sphinx(indent=0))
 
+    def get_charge_density(self):
+        """
+        Gets the charge density from the hdf5 file. This value is normalized by the volume
+
+        Returns:
+            pyiron.atomistics.volumetric.generic.VolumetricData
+        """
+        if self.status.finished not in [
+            "finished", "warning", "not_converged"
+        ]:
+            return
+        else:
+            with self.project_hdf5.open("output/generic") as ho:
```

Comment:
To be compatible with VASP maybe we should store this at `output` rather than `output/generic`

Reply:
Okay I moved it in the latest commits. Thanks again for pointing me to the VolumetricData class, it adds a lot of nice features.